docs/dependencies.md

```diff
@@ -54,18 +31,20 @@ In most cases, as a library author, you should not need to define any of these.
 The following settings are available on iOS and Android:
 
 ```ts
-type DependencyParamsIOST = {
+interface IOSDependencyParams {
   project?: string;
   podspecPath?: string;
-  sharedLibraries?: string[];
-};
+  scriptPhases?: Array<any>;
```

Comment:
any? would be nice to describe it somehow

Reply:
Yeah, it's been there like this in the code, mainly because it's not used inside JS but inside Ruby. Good call!